core/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "libtock-core"
+version = "0.1.0"
+authors = ["torfmaster <briefe@kebes.de>", "Woyten <woyten.tielesch@online.de>"]
```

Comment:
The authors list will likely become incorrect very quickly.

Reply:
I hope so, I'll fix that.

Reply:
fixed.